### ISSUE TITLE
Create a showcase of sites built with Backdrop

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#176]
We should start thinking about the showcase section of the backdropcms.org site. The following are todo items we'll need to build this out

- [ ] Look for example sites we would like to emulate, what do we like / dislike about them, and what would work well for Backdrop.
- [ ] Decide what information we would like to collect about each site
- [ ] Design a showcase (maybe?)
- [ ] Build the showcase content type & view
- [ ] Design review of the showcase (maybe?)
- [ ] Add a link into the main navigation

**Comment by wesruv** [comment]
http://www.awwwards.com/

Not a fan of the sites they showcase, but their site is slick.

**Comment by jenlampton** [comment]
Spreadsheet of content buckets from these various sites:
https://docs.google.com/a/jeneration.com/spreadsheets/d/14ZjraSla2o-goK7Tt-428ZhUBVfSELsCd4awt19qoEI/edit?usp=sharing

**Comment by cellear** [comment]
Are there nominations for sites we can showcase?  (I didn't have access to the spreadsheet above; I have requested access.)

**Comment by jenlampton** [comment]
@cellear any site can be added: https://backdropcms.org/node/add/showcase

**Comment by serundeputy** [comment]
@jenlampton I made a showcase node: https://backdropcms.org/showcase/journal-undergraduate-ethnography. It is not published feel free to publish if it helps for theming or anything.

**Comment by jenlampton** [comment]
Thanks @serundeputy ! :)

**Comment by Graham-72** [comment]
What should be the primary objective of our showcase? I am thinking that at this early stage the emphasis should be on the versatility of Backdrop to meet different needs and sizes of website, and the features available for site-building, both with core and with contributed modules.

It would seem too early to be a showcase of off-the-shelf themes but we can aim to demonstrate a range of different custom themes developed using the features available in Backdrop.

Also, we need to demonstrate practical examples of sites that have been migrated from Drupal to Backdrop.

We are not, I think, aiming to provide an art gallery of webdesign. Nor a sales brochure advertising the work of various webdesign agencies. Rather, we are aiming to demonstrate that Backdrop is worthy of consideration alongside other leading Content Management Systems.

These are my personal thoughts and I may well be wrong. I have just been reflecting on what I, as a web developer, would be looking for if I had not already chosen Backdrop.

**Comment by jenlampton** [comment]
Exactly :)

**Comment by Graham-72** [comment]
This being the case, I thought that http://www.drupal.org/case-studies was close to what we need. Perhaps a mock-up of a Backdrop version might be the next step, but we need more showcase sites.

**Comment by jenlampton** [comment]
We have a mockup, and I built out a layout and made the necessary theme changes last Thursday. Let me push that up to prod so you guys can take a look...

**Comment by jenlampton** [comment]
Here we go: https://backdropcms.org/showcase/americas-best-bootfitters

**Comment by jenlampton** [comment]
@serundeputy can you provide a screenshot that does not include the browser for https://backdropcms.org/showcase/journal-undergraduate-ethnography ?  I added one with CSS, too.

**Comment by Graham-72** [comment]
@jenlampton thanks for this. I like what I see. Two immediate problems I have as a contributor of sites

1. Getting the hero image right for different display widths.
2. Listing the modules that are used for key features. This is something I like to see when looking at sites and thinking 'how have they done that?'

**Comment by jenlampton** [comment]
2 will be added when we get a reference type module in core (so, hopefully Jan 15th?) then we can node-reference any project on the backdrop site already :)

for 1) would it be helpful to specify a recommended set of dimensions (like we do for phone & tablet?)

**Comment by Graham-72** [comment]
Yes it would be helpful to have recommended dimensions. Would we have the possibility to provide different hero images for the main different screen sizes? Another thought I have is that the style for the background image could center it perhaps.

**Comment by jenlampton** [comment]
I don't think it should center, since the text is left-justified. If it centered then what would work with the text on full-screen images may not work with the text on smaller screen sizes. If we Leave it left aligned then the same image is more likely to work on all devices.

There is something really weird about the positioning of the text though - I'll see if I can fix that now.

**Comment by Graham-72** [comment]
It is very difficult to choose one hero image that suits all circumstances,

**Comment by jenlampton** [comment]
This is not a unique situation. with the hero block in core we solve the problem with `background-size: cover;` which is what I've done here. I've also added the center alignment you recommended, and it seems to be working okay.

**Comment by Graham-72** [comment]
Thanks. Those changes seem to work well.
Can we get the text to show a bit more clearly where the background is light, perhaps by having a translucent grey background for the text, as often done with film subtitles?

**Comment by jenlampton** [comment]
that's a good idea! :) I was wondering how to handle that situation...

**Comment by serundeputy** [comment]
@jenlampton updated desktop image sans browser.

**Comment by jenlampton** [comment]
looks great, thanks @serundeputy !

@Graham-72 I added a gray background to the title/subtitle area. What do you think?
https://backdropcms.org/showcase/catherine-larner
https://backdropcms.org/showcase/deben-rowing-club

**Comment by mikemccaffrey** [comment]
> What should be the primary objective of our showcase?

I'm thinking that it should be showing the (hopefully) broad adoption of Backdrop. We want to show that is (hopefully) being used for lots of site, and that those sites (hopefully) look exciting and different from each other. 

Perhaps we should have a featured block with a really impressive example, followed by an endless scroll of additional sites that have been created.

**Comment by jenlampton** [comment]
Sounds great. But how are we going to get all that content?

**Comment by Graham-72** [comment]
> I added a gray background to the title/subtitle area. What do you think?

It certainly makes the text show up better.

But for smaller displays (i.e. phone) it looks a bit odd to have 90% (portrait) or 50% (landscape) of the hero image appearing darker. Perhaps for these sizes it would be best to make the whole hero image darker? Or just always use a darker image? @jenlampton how do you feel about the look of https://backdropcms.org/showcase/americas-best-bootfitters on a phone?

**Comment by jenlampton** [comment]
For me on mobile 100% of the hero image is darkened, only on tablet or desktop does the text bottom-align with the background behind it darkened.  Perhaps the CSS cache wasn't cleared last time you looked (sorry about that!). Can you check again?

**Comment by Graham-72** [comment]
Yes, its now looking fine on my mobile when in portrait.
When in landscape it is not so good (in my opinion), see screenshots (photos) below.
![americas-best-bootfitters500](https://cloud.githubusercontent.com/assets/10559274/19492417/7dd48b04-956e-11e6-8f86-e417bea598a8.jpg)
and
![catherinlarner500](https://cloud.githubusercontent.com/assets/10559274/19492482/c1225d5a-956e-11e6-8c6a-dbf184570b76.jpg)

**Comment by jenlampton** [comment]
I think your phone is larger than mine. It still looks good on my phone when in landscape. I used the same breakpoints for this as we use on the rest of the site.

I was thinking it might be time to change some things about the site, in particular 1) custom layouts 2) Menu handling 3) drawer handling.  Perhaps the fix on these babies can happen in conjunciton with that, adopting the standard breakpoints from bootstrap?

**Comment by Graham-72** [comment]
Understood. I do have great difficulty using the site navigation on a mobile so will welcome these changes.

Do you want any more sites added to the showcase?

**Comment by jenlampton** [comment]
Yes, I'd love to see as many as possible - assuming they look good and have compelling testimonials, of course.

**Comment by Graham-72** [comment]
How are we going to encourage others to add their sites to the Showcase? There must be several out there. Can people recommend, perhaps here, any they come across?

**Comment by olafgrabienski** [comment]
In the issue referenced by @jenlampton there was asked in a [comment](https://github.com/backdrop-ops/backdropcms.org/issues/316#issuecomment-332989215) if there is some styling missing from the Showcase page. I have the same question.

While the individual showcase pages look good, the overview page looks quite raw (see screenshot in the comment mentioned above). Do we still want to design the overview page like the screenshot in the [description of this issue](https://github.com/backdrop-ops/backdropcms.org/issues/176#issue-133072848)?

**Comment by jenlampton** [comment]
We have the design @olafgrabienski. Yesterday I posted a screenshot of it into the original issue above. This ticket is still open because we need to implement the design.  What's there now is temporary.

**Comment by BWPanda** [comment]
While we wait for a reference feature/module to get into core, can we use one of the existing contrib modules so we can add information about the people/companies who built each site?

**Comment by jenlampton** [comment]
Yeah, that's a good idea. I think we can use the reference module as it does most of what is usually needed. And where it doesn't we'll be motiviated to build that :)
